test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -61,7 +61,7 @@ def run_smdebug_test(
             f"{container_test_local_dir}:{os.path.join(os.sep, 'test')}{shm_setting}{image_uri} "
             f"./{test_script} {framework}",
             hide=True,
-            timeout=2400,
+            timeout=20000,
```

Comment:
The total build timeout for PRs is 1.5 hours. The test is still timing out, so looks to be a separate issue. We can perhaps skip it for TF1 if it's been verified to work locally